src/Microsoft.DotNet.Wpf/specs/tooltip.md

```diff
@@ -0,0 +1,305 @@
+
+## Mouse buffer zone
+When you open a tooltip by hovering the mouse over its parent element, you should be able to move the mouse onto the tooltip without closing the tooltip.
+To that end, WPF implicitly declares a *mouse buffer zone*: a rectangle that connects the parent element to the tooltip, in which you can move the mouse without closing the tooltip due to hover-end.
+(Note that the mouse still interacts in the usual way with elements in the buffer zone, and that interaction might close the tooltip.)
+
+After using the usual placement rules to determine the size and position of the tooltip, WPF calculates the buffer zone to be a rectangle with the following properties:
+- it contains the shortest straight line between the parent and its tooltip
+- it is large enough to accommodate reasonable drift as you move the mouse along that line
+- it is no larger than needed to satisfy the first two properties
+
+In the common case where the tooltip overlaps its parent, the buffer zone may be empty.
+
+It may be impossible or impractical to find a suitable rectangle, in which case the buffer zone is empty and moving the mouse off the parent element will close the tooltip.
+This only happens in extreme cases, for example when the tooltip and its parent are on different monitors. 
+
+
+# API Pages
+
+## ToolTipService.ShowsToolTipOnKeyboardFocus attached property
+Gets or sets whether an element shows its tooltip when it acquires keyboard focus.
+
+```
+see GetShowsToolTipOnKeyboardFocus, and SetShowsToolTipOnKeyboardFocus
+```
+
+### Remarks
+Setting this property on a parent element that has a tooltip helps control whether to open the tooltip when the element acquires focus by keyboard navigation.
+(Note: when the parent acquires focus by mouse-click or touch, the tooltip is not opened.)
+The tooltip is opened according to the following table, where the rows indicate the value of the `ToolTipService.ShowsToolTipOnKeyboardFocus` attached property on the parent element, and the columns indicate the value of the ToolTip element's `ShowsToolTipOnKeyboardFocus` property:
+
+|           | False | null  | True  |
+|-----------| ----- | ----  | ----  |
+| **False** | False | False | False |
+| **null**  | False | True  | True  |
+| **True**  | True  | True  | True  |
+
+If there is no ToolTip element, the 'null' column applies.
+
+In other words, honor an explicit value (True or False); if both the parent element and the ToolTip element have explicit values, the parent's value takes precedence;  if neither has an explicit value, open the tooltip.  The latter case is the default.
+
+## ToolTip.ShowsToolTipOnKeyboardFocus property
+Gets or sets whether the ToolTip is shown when its parent element acquires keyboard focus.
+
+```
+    public bool? ShowsToolTipOnKeyboardFocus { get; set; }
+```
+
+### Remarks
+Setting this property on ToolTip element helps control whether to open the tooltip when its parent element acquires focus by keyboard navigation.
```

Comment:
Is this the same description as the Service version? If so, please say "see ...", so I don't have to diff these two to find the differences :). When we copy this to docs.microsoft.com we can paste twice.

Reply:
They're the same.